scalding-core/src/main/scala/com/twitter/package.scala

```diff
@@ -34,7 +34,7 @@ package object scalding {
   /**
    * Make sure this is in sync with version.sbt
    */
-  val scaldingVersion: String = "0.17.2"
+  val scaldingVersion: String = "0.18.0"
```

Comment:
I believe this is the only place that we refer to scalding version within src. As mentioned above, this can be addressed with something like https://github.com/sbt/sbt-buildinfo. But considering that it is already out of date and clearly not critical, I think it would be fair for a followup CC @johnynek

Reply:
I have moved this change to https://github.com/twitter/scalding/pull/1976 as it is independent of what is being proposed here